### DISCUSSION TITLE
Different classes of registrations

### DISCUSSION THREAD

**Discussion by BobPusateri** [dataplat/DataSaturdays#66] [Ideas] [answered]
Assuming the tooling here will eventually support some sort of event registration, I'd love to see support for different classes of registrants, such as:

- attendees
- speakers
- sponsors

The SQL Saturday tools always annoyed me in that the attendance limit combined everyone, and if (and I do mean **if**) sponsors registered their people, it took spots away from attendees, or I had to adjust my maximum numbers to compensate. I think being able to at the very least separate out attendees for purposes of "maximum attendance" would give significant flexibility.

**Comment by amtwo** [comment, marked as answer]
The current vision is more about building integrations for existing tools that do these things well: ex) Eventbrite for attendees, Sessionize for Speakers, etc. 

I think we should keep in mind that Speakers & Sponsors are attendees for some purposes (ex, capacity for physical space, lunch, etc)... But you often want to segment other communications separately so that Speakers/Sponsors get their "special" instructions, without getting the "standard" instructions, which may not apply to them. 

Implementation details may vary, but this could be as simple as creating a "turnkey" way to ensure selected speakers from Sessionize get auto-registered into a special "Speaker" ticket type in EventBrite. (or some other mechanism completely)

**Reply by BobPusateri** [reply]
My apologies for not knowing the full details of the vision. I'll be sure to think things through more before offering suggestions in the future.

**Reply by way0utwest** [reply]
No need to apologize, or think through. That's why we have discussions

**Reply by amtwo** [reply]
What Steve said!

I didn't intend it to be an admonishment at all-- more of a "let me bring you up to speed"

**Comment by way0utwest** [comment]
I would agree with @amtwo here. The idea is to integrate with existing services here where possible, rather than building.

If someone wants to build this, great, but for now, I'd look to integrate with 
- Eventbrite (definitely)
- Meetup (possibly)
- ??
